res/values/strings_de.arb

```diff
@@ -0,0 +1,165 @@
+{
+  "title": "Szivacs Napló",
+  "title_full": "Szivacs Napló 2",
+  "version": "version: ",
+  "version_number": "2.0",
+  "made_by": "hergestellt von: ",
+  "boa": "BoA",
+  "made_with": "hergestellt mit: ",
+  "flutter": "Flutter",
+  "youtube": "YouTube",
+  "telegram": "Telegram",
+  "email": "eSzivacs@gmail.com",
+  "github": "GitHub",
+  "instagram": "Instagram",
+  "absent_title": "Abwesenheiten / Verspätungen",
+  "mode": "Modus: ",
+  "subject": "Fach: ",
+  "teacher": "Lehrer: ",
+  "absence_time": "Zeit der Abwesenheit: ",
+  "administration_time": "Zeitpunkt der Protokollierung: ",
+  "justification_state": "Stand der Rechtfertigung: ",
+  "justification_mode": "Modus der Rechtfertigung: ",
+  "delay_mins": "Verspetete Zeit: ",
+  "ok": "ok",
+  "statistics": "Statistik",
+  "parental_justification": "Rechtfertigungen durch Eltern: $amount St",
+  "all_absences": "Menge aller Abwesenheiten*: $amount Stunden",
+  "excluding_delay": "*Verspätungen nicht enthältet.",
+  "all_delay": "Menge aller Verspätungen: $amount Minuten",
+  "sure": "Sicher?",
+  "delete_confirmation": "Möchtest du $name löschen?",
+  "no": "nein",
+  "yes": "ja",
+  "accounts": "Konten",
+  "evaluations": "Noten",
+  "theme": "Thema: ",
```

Comment:
Tudom, hogy kicsivel voltak az eredetiek, de németül főnevet bűn nem nagybetűvel kezdeni.

Reply:
Igen, ennyit én is értek a nyelvhez :)